nilearn/datasets/atlas.py

```diff
@@ -782,3 +782,63 @@ def fetch_coords_dosenbach_2010(ordered_regions=True):
                   networks=out_csv['network'], description=fdescr)
 
     return Bunch(**params)
+
+
+def fetch_atlas_surf_destrieux(data_dir=None, url=None,
+                               resume=True, verbose=1):
+    """Download and load the Destrieux cortical atlas (dated 2009)
+        on the Freesurfer fsaverage5 surface
+
+    Parameters
+    ----------
+    data_dir: string, optional
+        Path of the data directory. Use to forec data storage in a non-
+        standard location. Default: None (meaning: default)
+    url: string, optional
+        Download URL of the dataset. Overwrite the default URL.
+
+    Returns
+    -------
+    data: sklearn.datasets.base.Bunch
+        dictionary-like object, contains:
+        - Cortical ROIs, lateralized or not (maps)
```

Comment:
Returns section needs to be updated. What we return here are path to annotations in .annot format.
I would update with the same as what you have written in destrieux_surface.rst file.